src/ui-services/notificationsService.js

```diff
@@ -21,6 +21,10 @@ angular.module('openshiftCommonUI').provider('NotificationsService', function()
       // notifications may already have an id that is not necessarily unique,
       // this is an explicitly unique id just for `track by` in templates
       notification.trackByID = _.uniqueId('notification-') + Date.now();
```

Comment:
fallback would cover all the notifications currently. Going forward, we could define `toast:true|| false`, `drawer:true|| false`

Reply:
In my WIP I added a notification.showToast true||false to show/hide toast notifications being thrown from adding a notification to the drawer.

Reply:
@benjaminapetersen @dtaylor113 It makes sense to me to be able to control toast and drawer behavior per notification. If the default is to toast and not put it in the drawer, maybe

`notification.skipToast` (default: `false`)
`notification.showInDrawer` (default: `false`)

Opinion?

Reply:
Hard to know what the default should be.  For quota notifications, they should go into the drawer, but not toasted.

Reply:
Also, should we allow toast: false and drawer: false  -would that result in a lost notification?

Reply:
@spadgett I'm generally good with that, so long as our iterations on the drawer don't shift our presumptions so much that some time in the future we are confused about which is the default & which is the overridden (thus making out defaults confusing).    

@dtaylor113 we'd prob catch that in review?

Reply:
> I'm generally good with that, so long as our iterations on the drawer don't shift our presumptions so much that some time in the future we are confused about which is the default & which is the overridden (thus making out defaults confusing).

Right that's why I'm suggesting the default be `false` always. If it's missing, it's `false`. Then it's less confusing.

If we changed the default, we'd probably change the property name. Since they're exceptions, it's not likely used in too many places.

Reply:
@dtaylor113 i guess we could check & `Logger.log` it if a notification is misconfigured.  I'm less concerned about it though, this doesn't seem like something that would slip under the radar.